### PR TITLE
Few tests, tox, and travis.

### LINKED ISSUE #27: Add travis.yml
With black and mypy and pytests with coverage

### LINKED ISSUE #40: Add travis cache to speedup tox

### PR DESCRIPTION
**Pull request by JulienPalard**
Hello tox. Hello travis.

Very very minimum tests, better that nothing.

Closes #27
Closes #40

### CODE REVIEW

**Comment by JulienPalard** [comment]
Also I like one-shot tox.ini and travis-ci.yml. SIGSLEEP for me.

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+deps = flake8
+commands = flake8 tests/ potodo/
+
+[testenv:black]
+deps = black
+commands = black --check --diff tests/ potodo/
+
+[testenv:mypy]
+deps = mypy
+commands = mypy --ignore-missing-imports potodo/
```
Comment (Seluj78) [inline tox.ini]: Can you add a todo here to add #strict?

**Comment by Seluj78** [comment]
Why aren't mypy and others ran on 37 and 36 ? Only in 38 ?

**Comment by Seluj78** [comment]
Also it'd be cool to add later a test case for Python 3.5 (cc @deronnax )

**Comment by JulienPalard** [comment]
> Why aren't mypy and others ran on 37 and 36 ? Only in 38 ?

Because it's useless, type checking, line lengths, variables names, and so on will not "fail more" or "succeed more" on other versions, but tests can pass for 3.6 and fail for 3.5 (f-strings typically).

**Comment by JulienPalard** [comment]
> Also it'd be cool to add later a test case for Python 3.5 (cc @deronnax )

Yes, but only after someone removes f-strings.

File: tox.ini (hunk 1 above)
Comment (JulienPalard) [reply tox.ini]: There's already an issue about it (https://github.com/Seluj78/Potodo/issues/30) I don't feel committing it as a comment.

**Comment by JulienPalard** [comment]
@Seluj78 ^

**Comment by Seluj78** [comment]
@JulienPalard I corrected in on my end 🤷‍♂ I'll handle the conflict

File: potodo/potodo.py [hunk 2]
```diff
@@ -173,7 +172,7 @@ def exec_potodo(
     :param counts: Render list with counts not percentage
     """
 
-    # Initialize the arguments
+    # initialize the arguments
```
Comment (awecx) [inline potodo/potodo.py]: ?

Comment (JulienPalard) [reply potodo/potodo.py]: Probably due to a typo or a find and replace ... dunno :] fixing it.

**Pull request merged**